raiden/tests/integration/transfer/test_mediatedtransfer.py

```diff
@@ -409,6 +412,9 @@ def run_test_mediated_transfer_calls_pfs(raiden_network, token_addresses):
         app0.raiden.mediate_mediated_transfer(locked_transfer)
         assert patched.call_count == 2
 
+        app0.raiden.config['services']['pathfinding_service_address'] = None
+        app0.raiden.config['services']['pathfinding_eth_address'] = None
```

Comment:
Is this done to reset the state for the following tests? If so, using `patch.dict` will make this more obvious and also leave the correct state when the test fails due to some exception between setting and resetting.

Reply:
Yes, good idea.